test/development/app-dir/missing-required-html-tags/index.test.ts

```diff
@@ -64,7 +64,7 @@ describe('app-dir - missing required html tags', () => {
     )
 
     await openRedbox(browser)
-    // TODO(NDX-768): Should show "missing tags" error
```

Comment:
Why was this removed? No assertion was added that shows this was fixed. Underlying issue was also not marked as done.

Reply:
Oops messed with another task in a different pr, this one should be kept.

Reply:
Will be restored in https://github.com/vercel/next.js/pull/75796